setup.py

```diff
@@ -15,5 +15,19 @@
     author_email='',
     url='https://github.com/ModellingWebLab/cellmlmanip',
     license=license_,
-    find_packages=find_packages(exclude=('tests', 'docs'))
+    packages=find_packages(exclude=('tests', 'docs')),
+    package_data={'cellmlmanip': ['cellml_units.txt']},
```

Comment:
Do we need this _and_ the manifest file? Maybe manifest file wasn't working because of typo below?

Reply:
not sure. some discussion about needing both, one for source distributions and the other for binary distributions